dvc/command/fsck.py

```diff
@@ -5,23 +5,22 @@
 from dvc.data_cloud import file_md5
 
 
-class FileFsckDep(object):
-    def __init__(self, dvc_file_name, md5, type_name, use_cache):
+class FsckFileDep(object):
+    def __init__(self, dvc_file_name, dep):
         self.dvc_file_name = dvc_file_name
-        self.md5 = md5
-        self.type_name = type_name
-        self.use_cache = use_cache
-
-    def checksum_msg(self, hardlink_md5):
-        stage_error = '!!!' if self.md5 and hardlink_md5 and self.md5 != hardlink_md5 else ''
-        return '{} {}'.format(self.md5, stage_error)
+        self.md5 = dep.md5
+        self.type_name = type(dep).__name__
+        self.use_cache = dep.use_cache
```

Comment:
it should be 'dep.use_cache if hasattr(dep, 'use_cache') else False', as Dependency class doesnt have use_cache property anymore.

Reply:
done

Reply:
Thanks!